packages/core/build/tokens.ts

```diff
@@ -17,14 +17,14 @@ const layout = {
     xxl: token(96),
   },
   grid: {
-    cols: token(12, { absolute: true }),
+    cols: token(12, { static: true }),
   },
   width: {
-    xs: token(576),
-    sm: token(768),
-    md: token(992),
-    lg: token(1200),
-    xl: token(1440),
+    xs: token('576px', { static: true }),
+    sm: token('768px', { static: true }),
+    md: token('992px', { static: true }),
+    lg: token('1200px', { static: true }),
+    xl: token('1440px', { static: true }),
```

Comment:
Static ensure the value is not transformed in the token build process. This is important for values that are computed in sass mixins or values that can be css vars in media queries.